client/my-sites/domains/domain-management/list/style.scss

```diff
@@ -133,43 +133,6 @@
 	}
 }
 
-.domain-management-list-item__content {
```

Comment:
These are old CSS styles that are no longer used (and creating confusion) from the old domain management. I did a quick cleanup, but I'm sure there's more stale ones.